lib/internal/util.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const prefix = '(node) ';
+const prefix = `(${process.release.name}:${process.pid}) `;
```

Comment:
`process.title` can be user-set. I do it myself in many applications. I think it disqualifies just because of that.